Sources/TuistServer/Client/ServerClientOutputWarningsMiddleware.swift

```diff
@@ -52,8 +42,10 @@ struct ServerClientOutputWarningsMiddleware: ClientMiddleware {
             throw CloudClientOutputWarningsMiddlewareError.invalidSchema
         }
 
-        let logger = ServiceContext.$current.get()?.logger
-        json.forEach { logger?.warning("\($0)") }
+        #if canImport(TuistSupport)
+            let logger = ServiceContext.$current.get()?.logger
```

Comment:
The logger is also not available outside of `TuistSupport` – something to fix in the future as we'll want logging in the iOS app as well.